cub/cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1424,7 +1424,7 @@ struct DispatchRadixSort : SelectedPolicy
 #endif
 
           auto onesweep_kernel = DeviceRadixSortOnesweepKernel<
-            MaxPolicyT,
+            typename PolicyHub::MaxPolicy,
```

Comment:
Nitpick: We are using this all over the place, maybe we should have an alias template for that?

Reply:
I think I would just prefer if dispatchers would use a local alias.